cpu/nrf52/vectors.c

```diff
@@ -96,58 +99,93 @@ WEAK_DEFAULT void isr_pwm3(void);
 extern void SWI2_EGU2_IRQHandler(void);
 #endif
 
-/* CPU specific interrupt vector table */
-ISR_VECTOR(1) const isr_t vector_cpu[CPU_IRQ_NUMOF] = {
-    isr_power_clock,       /* power_clock */
-    isr_radio,             /* radio */
-    isr_uart0,             /* uart0 */
-    isr_spi0_twi0,         /* spi0_twi0 */
-    isr_spi1_twi1,         /* spi1_twi1 */
-    isr_nfct,              /* nfct */
-    isr_gpiote,            /* gpiote */
-    isr_saadc,             /* adc */
-    isr_timer0,            /* timer0 */
-    isr_timer1,            /* timer1 */
-    isr_timer2,            /* timer2 */
-    isr_rtc0,              /* rtc0 */
-    isr_temp,              /* temp */
-    isr_rng,               /* rng */
-    isr_ecb,               /* ecb */
-    isr_ccm_aar,           /* ccm_aar */
-    isr_wdt,               /* wdt */
-    isr_rtc1,              /* rtc1 */
-    isr_qdec,              /* qdec */
-    isr_lpcomp,            /* lpcomp */
-    isr_swi0,              /* swi0 */
-    isr_swi1,              /* swi1 */
-#ifdef SOFTDEVICE_PRESENT
-    SWI2_EGU2_IRQHandler,  /* softdevice swi handler */
-#else
-    isr_swi2,              /* swi2 */
-#endif
-    isr_swi3,              /* swi3 */
-    isr_swi4,              /* swi4 */
-    isr_swi5,              /* swi5 */
-    isr_timer3,            /* timer 3 */
-    isr_timer4,            /* timer 4 */
-    isr_pwm0,              /* pwm 0 */
-    isr_pdm,               /* pdm */
-    (0UL),                  /* reserved */
-    (0UL),                  /* reserved */
-    isr_mwu,               /* mwu */
-    isr_pwm1,              /* pwm 1 */
-    isr_pwm2,              /* pwm 2 */
-    isr_spi2,              /* spi 2 */
-    isr_rtc2,              /* rtc 2 */
-    isr_i2s,               /* i2s */
-#ifdef CPU_MODEL_NRF52840XXAA
-    isr_fpu,               /* fpu */
-    isr_usbd,              /* usbc */
-    isr_uarte1,            /* uarte1 */
-    isr_qspi,              /* qspi */
-    isr_cryptocell,        /* cryptocell */
-    isr_spi3,              /* spi3 */
-    (0UL),                 /* reserved */
-    isr_pwm3,              /* pwm3 */
+#if defined(CPU_MODEL_NRF52811XXAA)
```

Comment:
Are they so differeent that it can't be handled in-line like for `CPU_MODEL_NRF52840XXAA`?
Please keep the indentation.

Reply:
Not that different. There are just a lot of features missing and the addresses are different, so is the order. 
[See page 18.](https://infocenter.nordicsemi.com/pdf/nRF52811_PS_v1.0.pdf)
If I would add a lof of additional ifdef to it, it would get really hard to read.

Reply:
Alright then keep them separate, but leave the original formatting in place.